src/cljs/athens/views/blocks/textarea_keydown.cljs

```diff
@@ -374,6 +374,9 @@
         [char-offset _]                  (get-end-points target)]
 
     (cond
+      (util/navigate-key? keys-destruct) (cond
```

Comment:
Need to handle the case for textarea navigation here because in mac navigation while in textarea does not work if we try to do this in listners. The issue is somehow `meta` keypress is not bubbled up and for now we (alex and me) could not figure out why is it so.